_includes/header.liquid

```diff
@@ -1,6 +1,7 @@
 <nav class="skip-to">
   <a href="#content">Skip to main content</a>
-  <a href="#nav">Skip to local navigation</a>
+  {% comment %} TODO is there a better way of deciding whether to show the local nav skip link? {% endcomment %}
```

Comment:
Sorry this is a bit of a hack - seemed the easiest way to determine whether the page would have a local nav.